app/buyers/views/buyers.py

```diff
@@ -319,6 +323,10 @@ def row(r):
         answers.update({'Day rate': r.get('dayRate', '')})
         answers.update(zip(ess_req_names, ess_responses))
         answers.update(zip(nth_req_names, nth_responses))
+
+        for q in answers.keys():
+            answers[q] = csv_cell_sanitize(answers[q])
```

Comment:
this is more succinct as: 

`answers = { k: csv_cell_sanitize(v) for k, v in answers.items() }`